src/stratis_cli/_actions/_top.py

```diff
@@ -652,11 +652,24 @@ def get_report(namespace):
         :raises StratisCliEngineError:
         """
         # pylint: disable=import-outside-toplevel
-        from ._data import Report
+        if (
```

Comment:
```report_name``` will always have some non-```None``` value, due to the parser change. So it's not necessary to check for it here.